### PR TITLE
Fixed all noted camelcase in function/variable/file names and made minor spelling edits

### PR DESCRIPTION
**Pull request by herinckc**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [ ] **If creating a new file :**
  - [ ] added links to it in the README files ?
  - [ ] included tests with it ?
  - [ ] added description (overview of algorithm, time and space compleixty, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [X] wrote short description in the PR explaining what the changes do ?
  - [X] Fixes #[issue number] if related to any issue

- [ ] **other**

I basically just looked over all files in the algorithms folder and changed all relevant camelcase to lowercase and underscore. It's important to note that I can't guarantee I removed every instance of camelcase, just the ones that I saw. I also made minor spelling changes here and there (i.e., occurance -> occurrence). These edits are in response to Issue #12.

### CODE REVIEW

File: algorithms/tree/bst/depth_sum.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def recur_depth_sum(root, n):
 """
 
 class TestSuite(unittest.TestCase):
-    def setUp(self):
+    def setup(self):
```
Comment (goswami-rahul) [inline algorithms/tree/bst/depth_sum.py]: actually `setUp` is the abstract method of `unittest.TestCase`, and we override it here. So, please it back to `setUp`. Thanks.

File: algorithms/stack/longest_abs_path.py [hunk 2]
```diff
@@ -48,14 +48,14 @@ def length_longest_path(input):
         print("stack: ", stack)
         print("curlen: ", currlen)
```
Comment (goswami-rahul) [inline algorithms/stack/longest_abs_path.py]: you missed this `currlen`.

**Comment by herinckc** [comment]
@goswami-rahul thanks for the quick response! I've added a couple new commits to address the concerns you've raised. Please let me know if I've missed anything else
